tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -342,6 +358,7 @@ var CESIUM_MAP_VIEW = (function() {
             else if (curr_entity_options.source.toLowerCase() == 'geojson')
             {
                 var gjson = curr_entity_options.options;
+                var default_point = gjson && gjson['properties'] && gjson['properties']['default_point'] == 'point'
```

Comment:
Change `default_point` to something more descriptive like `point_instead_of_billboard`.

Reply:
Also add a comment to explain why this is necessary.